conf/local-e2e-docker-config.json

```diff
@@ -34,6 +34,7 @@
   "salt_snapshot_expires_after_seconds": 180000,
   "is_auth_disabled": true,
   "enable_keysets": true,
+  "enable_s3_encryption": true,
```

Comment:
Best to keep all the s3 encryption config values together, so please move this to next to the other settings in all the config files